### DISCUSSION TITLE
Window size in headless mode maxes out at display resolution

### DISCUSSION THREAD

**Discussion by no1robino** [ultrafunkamsterdam/undetected-chromedriver#1171] [General]
If I run undetected-chromedriver in headless mode using 'options.add_argument('--headless')' and set the window size using 'window-size=1920,1080' and then take a screenshot, it is clear that the window size is not what I set it to. It appears to max out at the current display resolution set in my display settings on Windows 10. Anyone got a solution for this?

**Comment by deniskzt** [comment]
Try this solution:
brws.get("https://wikipedia.org/")
brws.set_window_size(1220,2600)
